generator/routing_generator.cpp

```diff
@@ -79,7 +79,9 @@ bool CheckBBoxCrossingBorder(m2::RegionD const & border, osrm::NodeData const &
   return any && !all;
 }
 
-void FindCrossNodes(osrm::NodeDataVectorT const & nodeData, gen::OsmID2FeatureID const & osm2ft, borders::CountriesContainerT const & m_countries, string const & countryName, routing::CrossRoutingContextWriter & crossContext)
+void FindCrossNodes(osrm::NodeDataVectorT const & nodeData, gen::OsmID2FeatureID const & osm2ft,
+                    borders::CountriesContainerT const & m_countries, string const & countryName,
```

Comment:
m_countries is a good name for class member, but here is a function param. 
it would be good, to rename it to countries.